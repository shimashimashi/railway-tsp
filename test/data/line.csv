line_cd,company_cd,line_name,line_name_k,line_name_h,line_color_c,line_color_t,line_type,lon,lat,zoom,e_status,e_sort
1,3,中央新幹線,チュウオウシンカンセン,中央新幹線,,,,137.49389648438,35.411438052435,8,1,1001
2,3,東海道新幹線,トウカイドウシンカンセン,東海道新幹線,,,,137.72148868199,35.144121935483,7,0,1002
3,4,山陽新幹線,サンヨウシンカンセン,山陽新幹線,,,,133.14789611678,34.419337699534,7,0,1003
4,2,東北新幹線,トウホクシンカンセン,東北新幹線,,,,140.76319219429,38.274266939284,7,0,1004