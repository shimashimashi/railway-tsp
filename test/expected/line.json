{"lines":[{"line_code":1,"line_name":"中央新幹線"},{"line_code":2,"line_name":"東海道新幹線"},{"line_code":3,"line_name":"山陽新幹線"},{"line_code":4,"line_name":"東北新幹線"}]}
